src/scenes/Projects/Files/useUploadProjectFiles.ts

```diff
@@ -92,29 +80,12 @@ export const useUploadBudgetFile = (): UploadFilesConsumerFunction => {
     uploadId,
     name,
     parentId: id,
-    action,
   }) => {
     await uploadFile({
       variables: {
         id,
         universalTemplateFile: { uploadId, name },
       },
```

Comment:
This hook should be able to go away. Can't we just pass the mutation document to the DefinedFileCard?